### PR TITLE
Migrate to GH actions

### PR DESCRIPTION
**Pull request by SilleBille**
This patch removes Travis CI and adds Github Actions as its
primary CI.

Example execution: https://github.com/SilleBille/ldap-sdk/actions/runs/86112694

### CODE REVIEW

File: .github/workflows/required.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+name: Required Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```
Comment (edewata) [inline .github/workflows/required.yml:9]: Just curious, does this mean when we create a branch we need to adjust these branch names? Or is it possible to get the current branch name from a variable?

**Review by edewata — APPROVED** [review]
Just one question, but it LGTM.

File: .github/workflows/required.yml (hunk 1 above)
Comment (SilleBille) [reply .github/workflows/required.yml:9]: With current code we need to adjust the branch names. There is no variable as such to get the branch name.

Removing the following code and adding `on: [push, pull_request]` triggers CI for push/PR against all branches

````
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
````

**Comment by SilleBille** [comment]
Merging as per the ACK. We can adjust the branch names in a future PR. Thanks!

**Pull request merged**